exercises/protein-translation/test/Tests.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import ProteinTranslation (toProtein)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "toProtein" $ for_ cases test
+  where
+    test Case{..} = it description $ toProtein strand `shouldBe` expected
+
+data Case = Case { description :: String
+                 , strand      :: String
+                 , expected    :: [String]
+                 }
+
+cases :: [Case]
+cases = [ Case { description = "identifies methionine codon"
+               , strand      = "AUG"
+               , expected    = ["Methionine"]
+               }
+        , Case { description = "identifies phenylalanine codon (UUU)"
+               , strand      = "UUU"
+               , expected    = ["Phenylalanine"]
+               }
+        , Case { description = "identifies phenylalanine codon (UUC)"
+               , strand      = "UUC"
+               , expected    = ["Phenylalanine"]
+               }
+        , Case { description = "identifies leucine codon (UUA)"
+               , strand      = "UUA"
+               , expected    = ["Leucine"]
+               }
+        , Case { description = "identifies leucine codon (UUG)"
+               , strand      = "UUG"
+               , expected    = ["Leucine"]
+               }
+        , Case { description = "identifies leucine codon (UUG)"
```

Comment:
is this case any different from the one immediately above? I see two `"identifies leucine codon (UUG)"`